app/api/controllers/RequestHandler.scala

```diff
@@ -18,7 +18,7 @@ package api.controllers
 
```

Comment:
Line 168 - "private def errorResult". This looks very similar to the errorResult method in BaseController.scala. Could this be unnecessary duplication?

Reply:
Good catch. Probably can remove the latter if it is unused.